tools/postman/validation/spectral.yaml

```diff
@@ -0,0 +1,89 @@
+functions:
+  [
+    folderDescription,
+    requestNameShouldNotEndInCopy,
+    patchRequestShouldHaveBody
+  ]
+
+rules:
+  collection-description-is-mandatory:
+    description: Collection must have a description.
+    given: $.collection.info
+    severity: error
+    then:
+      field: description
+      function: truthy
+
+  request-description-is-mandatory:
+    description: All requests must have a description.
+    given: $..request
+    severity: error
+    then:
+      field: description
+      function: truthy
+
+  folder-description-is-mandatory:
+    description: All folders should have a description.
+    message: "{{error}}"
+    given: $
+    severity: warn
+    then:
+      function: "folderDescription"
+
+  request-url-is-mandatory:
+    description: Requests must have a URL.
+    given: $..request
+    severity: error
+    then:
+      field: url
+      function: truthy
+
+  all-responses-have-status-code:
+    description: All responses must have a status code.
+    given: $..response[*]
+    severity: error
+    then:
+      field: code
+      function: truthy
+
+  http-should-not-be-used:
+    description: HTTP should not be used.
+    message: "HTTP should not be used: {{value}}"
+    given: $..url.raw
+    severity: warn
+    then:
+      function: pattern
+      functionOptions:
+        notMatch: "/^http:/i"
+        
+  request-name-should-not-end-in-Copy:
+    description: Request names should not end in "Copy".
+    message: "{{error}}"
+    given: $..item[*]
+    severity: error
+    then:
+      function: requestNameShouldNotEndInCopy
+
+  query-parameters-require-description:
+    description: All request query parameters must have a description.
+    given: $..request.url.query[*]
+    severity: error
+    then:
+      field: description
+      function: truthy
+
+  path-parameters-require-description:
+    description: All request path parameters must have a description.
+    given: $..request.url.variable[*]
+    severity: error
+    then:
+      field: description
+      function: truthy
```

Comment:
how are these different from https://github.com/mongodb/openapi/blob/0819d1d3d007c4498e10db1518b977b873bdd28b/.spectral.yaml#L41-L56

Reply:
There is no difference, I will remove them